examples/statdb-client/main.go

```diff
@@ -0,0 +1,159 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+
+	"go.uber.org/zap"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+
+	proto "storj.io/storj/pkg/statdb/proto"
+)
+
+var (
+	port   string
+	apiKey = []byte("abc123")
```

Comment:
same here: APIKey

Reply:
i didn't comment on all the APIKey stutff (cuz that's annoying) i'll let you change it in the files.